modules/mongodb/mongodb_test.go

```diff
@@ -56,6 +56,59 @@ func TestMongoDB(t *testing.T) {
 				mongodb.WithReplicaSet("rs"),
 			},
 		},
+		{
+			name: "With Auth, Replica set and mongo:7",
```

Comment:
suggestion: even though the previos test names are using sentences, where are moving away from them, instead using dashes and using and imperative style. Something like this:
```suggestion
			name: "with-auth/replica/mongo:7",
```

So please use this as reference in the new tests you added. And if you update the existing tests in this file, a separate commit for separation of concerns, that would be a perfect boy-scout trip!